CRM/UF/Page/ProfileEditor.php

```diff
@@ -20,18 +20,26 @@ public static function registerProfileScripts() {
 
     CRM_Core_Resources::singleton()
       ->addSettingsFactory(function () {
+        $ufGroups = civicrm_api3('UFGroup', 'get', array(
+          'sequential' => 1,
+          'is_active' => 1,
+          'rowCount' => 1000, // FIXME
+        ));
+        if (CRM_Core_Permission::check('manage event profiles') && !CRM_Core_Permission::check('administer CiviCRM')) {
+          foreach ($ufGroups['values'] as $key => $value) {
+            $ufJoin = CRM_Core_BAO_UFGroup::getUFJoinRecord($value['id']);
+            if (in_array('CiviEvent', $ufJoin)) {
+              $ufGroups['values'][$key]['module'] = 'CiviEvent';
+            }
+          }
+        }
         return array(
```

Comment:
created https://github.com/civicrm/civicrm-core/pull/6355 to handle 'CiviEvent_Additional'.